akka-stream/src/main/scala/akka/stream/javadsl/Tcp.scala

```diff
@@ -3,11 +3,14 @@
  */
 package akka.stream.javadsl
 
-import java.lang.{ Iterable ⇒ JIterable }
+import java.lang.{Iterable => JIterable}
 import java.util.Optional
-import akka.NotUsed
+
+import akka.{Done, NotUsed}
```

Comment:
huh so that's the formatting now?

Reply:
Nope, auto formatting didn't hit for some reason